beacon-chain/p2p/discovery_test.go

```diff
@@ -161,6 +161,7 @@ func TestStaticPeering_PeersAreAdded(t *testing.T) {
 	s.Start()
 	s.dv5Listener = &mockListener{}
 	defer s.Stop()
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Instead of sleep here, would `sync.WaitGroup` be better?

Reply:
wait group would only work if we could use it in each go-routine, however the method is enclosed in `s.Start()` so it isnt accesible to `sync.WaitGroup` from the test

Reply:
Hm I see. So no way to avoid the sleep then?

Reply:
Unfortunately, not. Especially for the current test.